ts/store/reducers/profile.ts

```diff
@@ -121,7 +146,7 @@ const reducer = (
             ...currentProfile,
             has_profile: true,
             email: newProfile.email,
-            is_email_enabled: newProfile.is_email_enabled,
+            is_email_enabled: newProfile.is_email_enabled === true,
```

Comment:
is it needed?

Reply:
being the `is_email_enabled` an optional attribute, it was aimed to get a boolean (false) even if it is `undefined`